torch/distributed/_tensor/op_schema.py

```diff
@@ -153,21 +153,13 @@ def output_mesh_shape(self):
             assert output_spec[0] is not None
             return output_spec[0].mesh.shape
 
-    @property
-    def output_ndim(self):
```

Comment:
Not needed in the PR but since we remove the output prefix, should we rename output_mesh_shape to mesh_shape for consistency?

Reply:
Ah good point, I missed that. Let me change that in this PR, as it only appears locally in this file.